tests/baselines/reference/findAllRefs_importType_exportEquals.baseline.jsonc

```diff
@@ -477,16 +477,16 @@
 // === findRenameLocations ===
 // === /a.ts ===
 // <|type /*RENAME*/[|TRENAME|] = number;|>
-// namespace T {
+// <|namespace [|TRENAME|] {
 //     export type U = string;
-// }
+// }|>
 // <|export = [|TRENAME|];|>
```

Comment:
Yeah, reading this, this change is absolutely correct. If I have:

```ts
type Foo = number
namespace Foo {
   // ...
}
export = Foo
```

You absolutely want a rename of `Foo` to rename _both_ the type and the namespace, as they are one thing that's clearly been exported on purpose.

Reply:
@jakebailey It looks correct to me as well. Thanks for your help.